pkg/cmd/package/upload/upload.go

```diff
@@ -29,24 +31,29 @@ const (
 	FlagAliasOverwrite     = "overwrite"
 	FlagAliasOverwriteMode = "overwritemode" // I keep forgetting the hyphen
 
-	// replace-existing deprected in the .NET CLI so not brought across
-	FlagUseDeltaCompression = "use-delta-compression" // this is not yet supported, but will be in future when we implement OctoDiff in go
+	FlagUseDeltaCompression = "use-delta-compression"
+	FlagAliasDelta          = "delta" // for less typing
+
+	// "replace-existing" flag deprecated in the .NET CLI so not brought across
 
 	FlagContinueOnError = "continue-on-error"
 )
 
 type UploadFlags struct {
-	Package             *flag.Flag[[]string]
-	OverwriteMode       *flag.Flag[string]
-	UseDeltaCompression *flag.Flag[bool]
+	Package       *flag.Flag[[]string]
+	OverwriteMode *flag.Flag[string]
+
+	// Note: string here because cobra doesn't handle bool(default=true) well.
+	// If this were a bool flag, and the user entered --use-delta-compression false, it would come out as true
+	UseDeltaCompression *flag.Flag[string]
 	ContinueOnError     *flag.Flag[bool]
 }
 
 func NewUploadFlags() *UploadFlags {
 	return &UploadFlags{
 		Package:             flag.New[[]string](FlagPackage, false),
 		OverwriteMode:       flag.New[string](FlagOverwriteMode, false),
-		UseDeltaCompression: flag.New[bool](FlagUseDeltaCompression, false),
+		UseDeltaCompression: flag.New[string](FlagUseDeltaCompression, false),
```

Comment:
if the default is true, should this be a negative boolean flag?
Not sure if I am a fan of the negative, but just a thought

Reply:
I've changed it to be off by default, so changes to a bool now and is more clear.